src/hooks/usePhaseData.ts

```diff
@@ -67,6 +68,18 @@ export const usePhaseData = (phaseEnum?: PhaseEnum | null): GetPhaseData => {
     [constants],
   );
 
+  const allPhaseTags = useMemo(() => {
+    const allTags: Set<string> = new Set();
+    constants?.phases?.forEach((phase) => {
```

Comment:
Where are you getting the value `constants` from?

You will need to either pass it in or do soemthing like this:
```
const constants = useApiConstants();
```

Reply:
It should be to prevent it from using an old version of `constants`